client/utils/account-fees.tsx

```diff
@@ -90,14 +90,19 @@ export const formatMethodFeesTooltip = (
 ): JSX.Element => {
 	if ( ! accountFees ) return <></>;
 	const currentBaseFee = getCurrentBaseFee( accountFees );
+	// If the current fee doesn't have a fixed or percentage rate, use the base fee's rate. Eg. when there is a promotional discount fee applied. Use this to calculate the total fee too.
+	const currentFeeWithBaseFallBack = currentBaseFee.percentage_rate
+		? currentBaseFee
+		: accountFees.base;
 
+	// Use the current fee with the base fallback, if applicable, to calculate the total fee. Eg. when we have a promotional discount fee applied.
```

Comment:
I believe we could omit this comment.

Reply:
Sorry, I missed removing the redundant comment while removing older unwanted code. I have now removed it here 
b50fb9c

Thank you. (Marking as resolved)